group-coordinator/src/main/java/org/apache/kafka/coordinator/group/consumer/ConsumerGroup.java

```diff
@@ -709,38 +713,53 @@ private static void maybeUpdateServerAssignors(
     /**
      * Updates the subscribed topic names count.
      *
+     * @param topicsImage The current metadata for all available topics.
      * @param oldMember The old member.
      * @param newMember The new member.
      */
     private void maybeUpdateSubscribedTopicNames(
-        ConsumerGroupMember oldMember,
-        ConsumerGroupMember newMember
+            TopicsImage topicsImage,
+            ConsumerGroupMember oldMember,
+            ConsumerGroupMember newMember
     ) {
-        maybeUpdateSubscribedTopicNames(subscribedTopicNames, oldMember, newMember);
+        maybeUpdateSubscribedTopicNames(topicsImage, subscribedTopicNames, oldMember, newMember);
     }
 
     /**
      * Updates the subscription count.
      *
+     * @param topicsImage            The current metadata for all available topics.
      * @param subscribedTopicCount  The map to update.
      * @param oldMember             The old member.
      * @param newMember             The new member.
      */
     private static void maybeUpdateSubscribedTopicNames(
+        TopicsImage topicsImage,
         Map<String, Integer> subscribedTopicCount,
         ConsumerGroupMember oldMember,
         ConsumerGroupMember newMember
     ) {
         if (oldMember != null) {
-            oldMember.subscribedTopicNames().forEach(topicName ->
-                subscribedTopicCount.compute(topicName, ConsumerGroup::decValue)
-            );
+            processMemberSubscribedTopics(oldMember, ConsumerGroup::decValue);
         }
-
         if (newMember != null) {
-            newMember.subscribedTopicNames().forEach(topicName ->
-                subscribedTopicCount.compute(topicName, ConsumerGroup::incValue)
-            );
+            processMemberSubscribedTopics(newMember, ConsumerGroup::incValue);
```

Comment:
I think this and https://github.com/apache/kafka/pull/14327/files#diff-74fddde890b14d5626cb56e3eda5feb9d09ae0a4f469901517fed7eb1b11f34fR743 will throw a compilation error because the method `processMemberSubscribedTopics` accepts 4 parameters and we are passing only 2. I see the Jenkins build is also failing due to the same reason.